src/platform-implementation-js/platform-implementation.ts

```diff
@@ -37,15 +34,17 @@ import isValidAppId from './lib/is-valid-app-id';
 // Some types
 import type { AppLogger } from './lib/logger';
 const loadedAppIds: Set<string> = new Set();
-const memberMap = new WeakMap();
 export type PiOpts = {
-  appName: string | null | undefined;
-  appIconUrl: string | null | undefined;
-  suppressAddonTitle?: string | null | undefined;
+  appName?: string | null;
+  appIconUrl?: string | null;
+  /**
+   * @deprecated Is this used?
+   */
+  appVersion?: string;
+  suppressAddonTitle?: string | null;
   VERSION: string;
   globalErrorLogging: boolean;
-  eventTracking: boolean;
-  inboxBeta: boolean;
+  eventTracking?: boolean;
```

Comment:
This shouldn't be optional. We take a `Partial<PiOpts>` from the user and construct a `PiOpts` object with defaults filling in for any missing required properties including this one.